src/fetch/payouts.py

```diff
@@ -229,9 +229,6 @@ class TokenConversion:
 
 def extend_payment_df(pdf: DataFrame, converter: TokenConversion) -> DataFrame:
     """
-    TODO: add type: `eth_to_token` is a function with signature (int -> int)
-          add type: `token_to_eth` is a function with signature (int -> int)
-                (one is the compositional the inverse of the other)
```

Comment:
This change is irrelevant, but was done earlier and should have been removed already.